doc/index.rst

```diff
@@ -49,10 +49,12 @@ See :doc:`development` on setting up your environment and workflow.
 .. code-block:: python
 
    import fedmsg
-   fedmsg.publish(topic='testing', modname='test', msg={
+   fedmsg.publish(name='mybus', topic='testing', modname='test', msg={
```

Comment:
Yes it is, however, adding that parameter makes the example code easier to have work (and explain).
Otherwise, we have to explain that the default name is `__main__.$(hostname -s)`, and that should be added as the entry in `endpoints.py` instead of something like `mybus`.